.github/workflows/CI.yml

```diff
@@ -82,9 +82,15 @@ jobs:
       - name: build
         run: cargo build --manifest-path ${{ matrix.conf.name }}/Cargo.toml --features "${{ matrix.conf.features }}"
         if: matrix.rust != 'nightly'
+      - name: build-serde
```

Comment:
i would instead add the serde feature to https://github.com/gtk-rs/gtk-rs-core/pull/1070/files#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R26, same for the windows ci file